src/sagemaker_sklearn_extension/preprocessing/encoders.py

```diff
@@ -552,7 +601,7 @@ def transform(self, X):
             Transformed input.
 
         """
-        X_int, X_mask = self._transform(X, handle_unknown="unknown")
+        X_int, X_mask = self._transform(X, handle_unknown="ignore")
```

Comment:
I believe "ignore" will still cause an exception here. I did a quick sanity check where one categorical column meets the threshold set but another categorical column doesn't. Here's what i get
```
>>> X = [['Male', 1], ['Female', 3], ['Female', 2]]
>>> r = RobustOrdinalEncoder(threshold=2)
>>> r.fit(X)
/Users/wuwilton/Developer/github/sagemaker-scikit-learn-extension/src/sagemaker_sklearn_extension/preprocessing/encoders.py:579: UserWarning: feature at index 1 does not have any categories appearing more than 2 times
  i, threshold, "time" if threshold == 1 else "times"
RobustOrdinalEncoder(categories=[array(['Female'], dtype=object),
                                 array([], dtype=object)],
                     threshold=2)
>>> r.transform([['Female', 3], ['Male', 1], ['Other', 15]])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/wuwilton/Developer/github/sagemaker-scikit-learn-extension/src/sagemaker_sklearn_extension/preprocessing/encoders.py", line 604, in transform
    X_int, X_mask = self._transform(X, handle_unknown="ignore")
  File "/Users/wuwilton/miniconda3/lib/python3.7/site-packages/sklearn/preprocessing/_encoders.py", line 138, in _transform
    Xi[~valid_mask] = self.categories_[i][0]
IndexError: index 0 is out of bounds for axis 0 with size 0
```

Reply:
Also where you get "ignore"? Looking at the [_BaseEstimator](https://github.com/scikit-learn/scikit-learn/blob/0fb307bf3/sklearn/preprocessing/_encoders.py#L22) I only see "error" being checked.

Reply:
The `RobustOrdinalEncoder` states the possible values for `handle_unknown` are `{'error', 'ignore'}`. From briefly going through the code you are correct, only `error` is being checked. However, I changed the value from `unknown` to `ignore` to be consistent with the documentation. See documentation [here](https://github.com/scikit-learn/scikit-learn/blob/0fb307bf3/sklearn/preprocessing/_encoders.py#L217).

Reply:
The error above is due to the empty list that is at `self.categories[i]` where `i` is a feature where no categories meet the threshold. I have modified the code to handle this issue similar to how `ThresholdOneHotEncoder` handles it.